src/Components/VimeoVideo.test.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+
+import VimeoVideo from './VimeoVideo';
+import { VIMEO_VIDEO_ID } from '../Fixtures';
+
+describe('<VimeoVideo />', () => {
+  it('renders correctly for a video from vimeo', () => {
```

Comment:
I'm going to give it a try

I found this library which seems to do the job and be widely used

Going to use the recording feature to generate the mocks since video content can be pretty awful to write : https://github.com/nock/nock#recording
Then going to use the mocks
Then going to disable network requests for the whole test suite to make sure any call to the outer world is mocked : https://github.com/nock/nock#enabledisable-real-http-requests

Reply:
It looks like my test works, except on node.js v12...

Reply:
Looks ok now
Can someone review the tests I wrote?